transport_nantes/mobilito/views.py

```diff
@@ -282,19 +283,8 @@ def post(self, request: HttpRequest, *args, **kwargs) \
                 f"id={request.session.get('mobilito_session_id', '')}, "
                 f"user={request.user.email}, {e}")
 
-        # Thank you page
-        # request.session is used to fill the thank you page
-        request.session['recording_duration_minutes'] = int(
-            (
-                datetime.now(timezone.utc)
-                - mobilito_session.start_timestamp
-            ).total_seconds() // 60
-        )
-        request.session['number_of_pedestrians'] = number_of_pedestrians
-        request.session['number_of_bicycles'] = number_of_bicycles
-        request.session['number_of_cars'] = number_of_cars
-        request.session['number_of_public_transports'] = \
-            number_of_public_transports
+        messages.add_message(
+            request, messages.SUCCESS, 'Merci de votre participation !')
```

Comment:
Do you think you could think of something more dry and corporate?  ;-)

I'm not good at this in French.  Do you have any fun friends who could suggest a phrase?

Reply:
I'm doing either cheesy or Puppy on acid texts, or dry corporate, there is no in-between :laughing:

You're underselling yourself, you do have a quite good french, only some misses on spelling but nothing to be ashamed of. You do speak it probably better than my french-born-and-raised-in-France dad to be honest :laughing:

Reply:
Good thing your dad doesn't hang out on github. ;-)